### ISSUE TITLE
Smarter injection of stylesheet

### ISSUE THREAD

**Issue by RoelN** [arrowtype/type-x#110]
With every change to the font's settings, a new stylesheet is injected, including all the relevant fonts as base64 string. This happens for _all_ tabs. This can quickly get out of hand, as each addition adds the fonts as base64 string to the page (and thus the browser memory)

**Comment by simoncozens** [comment]
Yes, we only inject two stylesheets now. (Font file definitions and CSS styles.)
